compaction/compaction_manager.cc

```diff
@@ -789,7 +789,9 @@ future<> compaction_manager::stop_tasks(std::vector<shared_ptr<task>> tasks, sst
     });
 }
 
-future<> compaction_manager::stop_ongoing_compactions(sstring reason, compaction::table_state* t, std::optional<sstables::compaction_type> type_opt) {
+future<> compaction_manager::stop_ongoing_compactions(sstring reason, compaction::table_state* t, std::optional<sstables::compaction_type> type_opt) noexcept {
+  // FIXME: restore indentation
```

Comment:
let's either fix the indentation in this series, or drop the comment, if we can live with half-indent here :)

Reply:
Will add a patch to restore indentation.